src/Symfony/Bridge/Twig/Tests/Extension/DumpExtensionTest.php

```diff
@@ -22,7 +23,7 @@ class DumpExtensionTest extends \PHPUnit_Framework_TestCase
      */
     public function testDumpTag($template, $debug, $expectedOutput, $expectedDumped)
     {
-        $extension = new DumpExtension(new VarCloner());
+        $extension = new DumpExtension(new VarCloner(), new HtmlDumper());
```

Comment:
This change should be reverted to check that the expected behaviour does not change when still not injecting a dumper. You could add another test which gets another dumper injected and test for the expected output.